README.md

```diff
@@ -112,133 +106,15 @@ npm run build
 # open developer console
 ```
 
-## Endpoint support
-
-*Starknet endpoints* (in compliance with [Starknet specs](https://github.com/starkware-libs/starknet-specs)):
-
-| Endpoint                                   | Supported          |
-|:-------------------------------------------| :----------------- |
-| `starknet_specVersion`                     | :white_check_mark: |
-| `starknet_getBlockWithTxHashes`            | :white_check_mark: |
-| `starknet_getBlockWithTxs`                 | :white_check_mark: |
-| `starknet_getStateUpdate`                  | :white_check_mark: |
-| `starknet_getStorageAt`                    | :white_check_mark: |
-| `starknet_getTransactionByHash`            | :white_check_mark: |
-| `starknet_getTransactionStatus`            | :white_check_mark: |
-| `starknet_getTransactionByBlockIdAndIndex` | :white_check_mark: |
-| `starknet_getTransactionReceipt`           | :white_check_mark: |
-| `starknet_getClass`                        | :white_check_mark: |
-| `starknet_getClassHashAt`                  | :white_check_mark: |
-| `starknet_getClassAt`                      | :white_check_mark: |
-| `starknet_getBlockTransactionCount`        | :white_check_mark: |
-| `starknet_call`                            | :white_check_mark: |
-| `starknet_estimateFee`                     | :white_check_mark: |
-| `starknet_estimateFeeSingle`               | :white_check_mark: |
-| `starknet_blockNumber`                     | :white_check_mark: |
-| `starknet_blockHashAndNumber`              | :white_check_mark: |
-| `starknet_chainId`                         | :white_check_mark: |
-| `starknet_syncing`                         | :white_check_mark: |
-| `starknet_syncing`                         | :white_check_mark: |
-| `starknet_getEvents`(not validated)        | :white_check_mark: |
-| `starknet_getNonce`                        | :white_check_mark: |
-| `starknet_addDeclareTransaction`           | :x:                |
-| `starknet_addDeployAccountTransaction`     | :x:                |
-| `starknet_addInvokeTransaction`            | :x:                |
-| `starknet_getProof`                        | :white_check_mark: |
-| `starknet_getBalance`                      | :white_check_mark: |
-| `starknet_getStateRoot`                    | :white_check_mark: |
-
-## Support
-
-Reach out to the maintainer at one of the following places:
-
-- [GitHub Discussions](https://github.com/keep-starknet-strange/beerus/discussions)
-- Contact options listed on
-  [this GitHub profile](https://github.com/keep-starknet-strange)
-
 ## Security
 
 Beerus follows good practices of security, but 100% security cannot be assured.
 Beerus is provided **"as is"** without any **warranty**. Use at your own risk.
 
-For more information and to report security issues, please refer to our
-[security documentation](docs/SECURITY.md).
-
 ## Acknowledgements
 
 - Huge props to A16z for their work on
   [helios](https://github.com/a16z/helios/).
 
-## Contributors ✨
```

Comment:
I'd rather keep this as an acknowledgement of previous contributions, at least until we bring something of major value to the table.